Geometry/HGCalCommonData/data/dd4hep/cms-test-ddhgcalwafer8-algorithm.xml

```diff
@@ -27,7 +27,6 @@
     <Include ref="Geometry/HGCalCommonData/data/hgcalcell/v9/hgcalcell.xml"/>
     <Include ref="Geometry/HGCalCommonData/data/hgcalwafer/v9/hgcalwafer.xml"/>
     <Include ref="Geometry/HGCalCommonData/data/hgcalwafer/v9/hgcalpos.xml"/>
-    <Include ref="Geometry/HGCalCommonData/data/dd4hep/world.xml"/>
```

Comment:
Why the random change in this unrelated file??

Reply:
Not sure how this was pushed into the commit. Will remove this.

Reply:
Fixed.